custom-commands/smart-room-lite/README.md

```diff
@@ -0,0 +1,11 @@
+# Smart Room Lite - A Custom Commands application
+
+## Purpose
+This application is built with the goal of being used as a quick-start for documentation hosted on docs.microsoft.com
```

Comment:
docs.microsoft.com should be a link to the docs you are referring to

Reply:
I have changed ref to point to the currently hosted documentation. We will need to revisit this, once we have the new documentation changes in -- cause the link address would change at that point.